src/styles/feature-item.scss

```diff
@@ -0,0 +1,9 @@
+.feature-link {
+  margin-left: -40px;
```

Comment:
Let's avoid using negative margins for things like this (and in other cases you've used negative margins). Instead you can set the padding on `.category-list` to 0.